cli/tsc/dts/lib.deno_url.d.ts

```diff
@@ -408,34 +424,112 @@ interface URL {
+   *
+   * // Creating a URL from a relative URL string with a base URL
+   * const url2 = new URL('/bar', 'https://example.org');
+   * console.log(url2.href);  // Logs "https://example.org/bar"
+   * ```
+   *
    * @see https://developer.mozilla.org/docs/Web/API/URL/URL
    */
   new (url: string | URL, base?: string | URL): URL;
 
   /**
+   * Parses a URL string or URL object and returns a URL object.
+   *
+   * @example
+   * ```ts
+   * const myURL = URL.parse('https://example.org');
+   * console.log(myURL.href);  // Logs "https://example.org/"
+   * console.log(myURL.hostname);  // Logs "example.org"
+   * console.log(myURL.pathname);  // Logs "/"
+   * console.log(myURL.protocol);  // Logs "https:"
+   *
+   * const baseURL = new URL('https://example.org');
+   * const myNewURL = URL.parse('/foo', baseURL);
+   * console.log(myNewURL.href);  // Logs "https://example.org/foo"
+   * console.log(myNewURL.hostname);  // Logs "example.org"
+   * console.log(myNewURL.pathname);  // Logs "/foo"
+   * console.log(myNewURL.protocol);  // Logs "https:"
+   * ```
+   *
    * @see https://developer.mozilla.org/docs/Web/API/URL/parse_static
    */
   parse(url: string | URL, base?: string | URL): URL | null;
 
   /**
+   * Returns a boolean value indicating if a URL string is valid and can be parsed.
+   *
+   * @example
+   * ```ts
+   * // Check if an absolute URL string is valid
+   * console.log(URL.canParse('https://example.org'));  // Logs true
+   * console.log(URL.canParse('https:://example.org'));  // Logs false
+   *
+   * // Check if a relative URL string with a base is valid
+   * console.log(URL.canParse('/foo', 'https://example.org'));  // Logs true
+   * console.log(URL.canParse('/foo', 'https:://example.org'));  // Logs false
+   * ```
+   *
    * @see https://developer.mozilla.org/docs/Web/API/URL/canParse_static
    */
   canParse(url: string | URL, base?: string | URL): boolean;
 
   /**
-   * @see https://developer.mozilla.org/docs/Web/API/URL/createObjectURL
+   * Creates a DOMString containing a URL representing the object given in the parameter.
+   * The URL lifetime is tied to the document in the window on which it was created.
+   *
+   * Returns a string containing an object URL that can be used to reference the contents of the specified source object.
+   *
+   * @example
+   * ```ts
+   * // Create a URL string for a Blob
+   * const blob = new Blob(["Hello, world!"], { type: "text/plain" });
+   * const url = URL.createObjectURL(blob);
+   * console.log(url);  // Logs something like "blob:null/1234-5678-9101-1121"
+   * ```
+   *
+   * The URL.createObjectURL() method is used to create a unique URL string that represents a given object, typically used for Blob, File, or MediaSource objects.
+   * For example it could be used to dynamically create Worker or SharedWorker objects, or to create a URL when a user uploads or creates a file, or to save canvas content in the browser.
```

Comment:
might be nice to have [the example from Kitson](https://github.com/denoland/deno/issues/9210) here
```
const src = new Blob([`console.log("hello deno");`]);
new Worker(URL.createObjectURL(src), { type: "module" });
```